src/main/java/com/jcabi/github/mock/MkReleases.java

```diff
@@ -121,6 +123,33 @@ public Release get(final int number) {
     }
 
     @Override
+    public Release latest() throws IOException {
+        Release latestRelease = null;
```

Comment:
@weisebrazil variables should be all lower case and non-compound, here you can use `latest` as the name